src/tailscale/error.ts

```diff
@@ -43,6 +43,11 @@ export function errorForType(type: string): TailscaleError {
           },
         ],
       };
+    case 'REQUIRES_RESTART':
```

Comment:
Mind if we change this to `FLATPACK_REQUIRES_RESTART`? It will make it easier to grok in the future.